Demo/Application/Base/BraintreeDemoContainmentViewController.m

```diff
@@ -64,6 +65,14 @@ - (void)setLatestTokenizedPayment:(id)latestPaymentMethodOrNonce {
     }
 }
 
+- (void)setLatestTokenizedPaymentString:(NSString*)latestNonceString {
+    _latestTokenizedPaymentString = latestNonceString;
```

Comment:
any reason you chose to use `_` to access the variable instead of `self` ? Seems like a typical setter through `self` would be okay here, unless I'm missing a BT requirement/piece of funcationality

Reply:
Nope, not a specific requirement. Used the `_` instead of `self` to match the above `setLatestTokenizedPayment` method that is used for the `BTPaymentMethodNonce` type. Since we don't have access to that here (currently) we just matched the method implementation. Still hoping we can resolve that issue and remove this function if we are able to get the Swift class to play nice with `BTPaymentMethodNonce`. The demo app as a whole seems to use both, so we can certainly update to use self if we feel strongly.